client/containers/Pub/PubSubmissionHeader/spubHeader.scss

```diff
@@ -0,0 +1,43 @@
+.spub-header-component {
+	border-bottom:	border-bottom: 1px solid #d2d2d2;
+;
+	padding: 200px;
+
+	h1 {
+		margin: 0;
+
+		font-size: 40px;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 56px;
+		letter-spacing: 0em;
+		text-align: left;
+
+		box-decoration-break: clone;
+	}
+
+	.tabs{
+		color: var(--community-accent-dark);
+		max-width: 900px;
+		> .bp3-tab-list {
+			position: absolute;
+			width: 1005.79px;
+			height: 0px;
+			padding-top: 5px;
+			.tab {
+				color: var(--community-accent-dark);
```

Comment:
The `text-align`, `font-size`, and `font-style` properties here don't need to be specified because they're already either the defaults, or provided by Blueprint.